content/zh/docs/reference/glossary/annotation.md

```diff
@@ -40,5 +40,5 @@ tags:
 The metadata in an annotation can be small or large, structured or unstructured, and can include characters not permitted by {{< glossary_tooltip text="labels" term_id="label" >}}. Clients such as tools and libraries can retrieve this metadata.
 -->
 
-注解中的元数据可大可小，可以是结构化的也可以是非结构化的，并且能包含{{< glossary_tooltip text="标签" term_id="label" >}}不允许使用的字符。像工具和软件库这样的客户端可以检索这些元数据。
+注解中的元数据可大可小，可以是结构化的也可以是非结构化的，并且能包含 {{< glossary_tooltip text="标签" term_id="label" >}} 不允许使用的字符。像工具和软件库这样的客户端可以检索这些元数据。
```

Comment:
No. We don't need these extra spaces around glossary_tooltip because the `text` in it is Chinese. We do not need extra spaces between Chinese characters.